appcast.xml

```diff
@@ -5,11 +5,24 @@
     <link>https://transmissionbt.com/appcast.xml</link>
     <description>Most recent version available.</description>
     <language>en</language>
-    <pubDate>Fri, 15 May 2020 20:00:00 -0400</pubDate>
+    <pubDate>Tue, 07 Feb 2023 20:00:00 -0400</pubDate>
     
     <item>
-      	<title>Transmission 3.00</title>
+      	<title>Transmission 4.00</title>
       	<sparkle:releaseNotesLink>https://transmissionbt.com/appcast/releasenotes.html</sparkle:releaseNotesLink>
+      	<pubDate>Tue, 07 Feb 2023 20:00:00 -0400</pubDate>
+      	<enclosure
+      	url="https://github.com/transmission/transmission-releases/raw/master/Transmission-4.00.dmg"
```

Comment:
This 404s for me

https://github.com/transmission/transmission/releases/download/4.0.0/Transmission-4.0.0.dmg

Reply:
Updated!